test/SILGen/win_bool_bridging.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: %target-swift-frontend -typecheck %s -I %clang-importer-sdk-path/usr/include -verify
+// REQUIRES: OS=windows-msvc
```

Comment:
Instead of a REQUIRES here, can you make a fake overlay and use an explicit triple? That way we can make sure not to break this from other platforms.

Reply:
I was trying to do that, but, I could not get enough of `Bool` to make that work.  I end up having to implement `Bool`, `Int8`, the operator groups, etc.

Reply:
Hm, right. We probably should have a way to do that at some point, but right now we don't. All right, never mind.